src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalRent = days * 40;
```

Comment:
I think it will be better to create variables for discount and basic price (Cheklist)

Reply:
> I think it will be better to create variables for discount and basic price (Cheklist)

thanks for your review :)